templates/node/src/Payload.ts.twig

```diff
@@ -0,0 +1,58 @@
+import { readFileSync } from "fs";
+import { File } from "node-fetch-native-with-agent";
+import { basename } from "path";
+
+export class Payload extends File {
+  private data: Buffer;
+  private fileName: string;
+
+  constructor(data: Buffer, name?: string) {
+    super([data], name || 'unnamed');
+    this.data = data;
+    this.fileName = name || 'unnamed';
+  }
+
+  public getData(): Buffer {
+    return this.data;
+  }
+
+  public getName(): string {
```

Comment:
I dont see this method being used. Can you link me where you used it?

Reply:
It is used in chunkedUpload